src/app/components/pages/Events.tsx

```diff
@@ -64,134 +65,158 @@ export const Events = withRouter(({ history, location }: RouteComponentProps) =>
-                        dispatch(
-                          getEventsList(numberOfLoadedEvents, EVENTS_PER_PAGE, typeFilter, statusFilter, stageFilter),
-                        );
-                      }}
-                    >
-                      Load more events
-                    </RS.Button>
-                  </div>
-                )}
+                  {/* Load More Button */}
+                  {numberOfLoadedEvents < total && (
+                    <div className="text-center mb-5">
+                      <RS.Button
+                        onClick={() => {
+                          dispatch(
+                            getEventsList(numberOfLoadedEvents, EVENTS_PER_PAGE, typeFilter, statusFilter, stageFilter),
+                          );
+                        }}
+                      >
+                        Load more events
+                      </RS.Button>
+                    </div>
+                  )}
 
-                {/* No Results */}
-                {total === 0 && (
-                  <div className="text-center">
-                    <p>Sorry, we cannot find any events that match your filter settings.</p>
-                    {statusFilter === EventStatusFilter["My booked events"] && (
-                      <p>
-                        N.B. Events booked via Eventbrite may not appear here; for these if you have received email
-                        confirmation you are booked.
-                      </p>
-                    )}
-                  </div>
-                )}
-              </div>
-            )}
-          />
-          <div className="mb-5">
-            <PageFragment fragmentId="event_type_descriptions" ifNotFound={RenderNothing} />
+                  {/* No Results */}
+                  {total === 0 && (
+                    <div className="text-center">
+                      <p>Sorry, we cannot find any events that match your filter settings.</p>
+                      {statusFilter === EventStatusFilter["My booked events"] && (
+                        <p>
+                          N.B. Events booked via Eventbrite may not appear here; for these if you have received email
+                          confirmation you are booked.
+                        </p>
+                      )}
+                    </div>
+                  )}
+                </div>
+              )}
+            />
+            <div className="mb-5">
+              <PageFragment fragmentId="event_type_descriptions" ifNotFound={RenderNothing} />
+            </div>
           </div>
-        </div>
-      </RS.Container>
-    </div>
+        </RS.Container>
+      </div>
+      <Banner
+        id="teacher-courses"
+        title="Computing events for teachers"
+        subtitle="Looking for teacher events?"
+        src="/assets/ncce-teachers.png"
+        link="https://teachcomputing.org/courses"
+        alt="teachers on a course"
+      >
+        {teacherEventsDescription}
```

Comment:
I think it would feel a bit more natural if this variable was a React element so that you can call it like this: `<TeacherEventsDescription />`.

Reply:
I figured it didn't really need to be a functional component since it doesn't actually *do* anything, just returns a bit of JSX with no logic that would involve it being re-rendered, but I don't think it makes any great difference. Switched it for readability!